File: js/04-heranca/main.js
import { Cidade } from "./Classes/Cidade/Cidade.js";
import { Fisica } from "./Classes/Pessoa/Fisica.js";
import { Juridica } from "./Classes/Pessoa/Juridica.js";

const c1 = new Cidade(1, 'Porto Alegre');
const c2 = new Cidade(2, 'Capão da Canoa');

const joao = new Fisica(1, 'João', '3368-5321', c1, '012-432-120-32');
const maria = new Fisica(2, 'Maria', '5321-3368', c2, '432-120-012-32');
const jose = new Fisica(3, 'Jose', '5321-3368', c2, '142-302-130-22');

const dosul = new Juridica(1, 'Supermercado Dosul', '9864-9012', c2, '03-123-245/0001-00');


dosul.addEmployee(maria);
dosul.addEmployee(jose);

dosul.logStatus();

File: py/04-heranca/main.py
from Classes.Pessoa.Pessoa import Pessoa
from Classes.Pessoa.Fisica import Fisica
from Classes.Pessoa.Juridica import Juridica
from Classes.Cidade.Cidade import Cidade

c1 = Cidade(1, 'Porto Alegre')
c2 = Cidade(2, 'Capão da Canoa')

joao = Fisica('João', '3368-5321', c1, '012-432-120-32')
maria = Fisica('Maria', '5321-3368', c2, '432-120-012-32')
jose = Fisica('Jose', '5321-3368', c2, '142-302-130-22')

dosul = Juridica('Supermercado Dosul', '9864-9012', '03-123-245/0001-00', c2)


dosul.printStatus()


dosul.addEmployee(joao)
dosul.addEmployee(jose)
dosul.printStatus()
